validator/src/bootstrap.rs

```diff
@@ -498,18 +510,32 @@ fn get_vetted_rpc_nodes(
                         rpc_contact_info.id,
                         rpc_contact_info.rpc()
                     );
+
+                    let rpc_addr = rpc_contact_info.rpc().ok()?;
+                    let ping_time = ping(&rpc_addr); 
+
                     let rpc_client = RpcClient::new_socket_with_timeout(
-                        rpc_contact_info.rpc().ok()?,
+                        rpc_addr,
                         Duration::from_secs(5),
                     );
 
-                    Some((rpc_contact_info, snapshot_hash, rpc_client))
+                    Some((rpc_contact_info, snapshot_hash, rpc_client, ping_time))
                 })
-                .filter(|(rpc_contact_info, _snapshot_hash, rpc_client)| {
+                .filter(|(rpc_contact_info, _snapshot_hash, rpc_client, ping_time)| {
                     match rpc_client.get_version() {
                         Ok(rpc_version) => {
-                            info!("RPC node version: {}", rpc_version.solana_core);
-                            true
+                            if ping_time.is_some() {
+                                info!("RPC node version: {} Ping: {:?}", rpc_version.solana_core, ping_time.unwrap());
```

Comment:
We should probably convert `ping_time` to milliseconds and include units in the info log.
`ping_time.as_millis()`